notification-service/src/constants/SUBSCRIPTIONS.ts

```diff
@@ -12,12 +12,11 @@ export const SUBSCRIPTIONS: EventSubscription[] = [
         },
     },
     {
-        id: 'MCD_DAI_Transfer',
-        address: '0x4f96fe3b7a6cf9725f59d353f723c1bdb64ca6aa',
+        id: 'MCD_DAI',
+        address: '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
```

Comment:
Will you point to the WETH in the end or will keep this one? Since you edited the `id`, you still need to change the mentioning of this id in the readme

Reply:
I will keep this one, as it calls the event very often!